### PR TITLE
Folder init and Login/sign up page

### PR DESCRIPTION
**Pull request by Vageshwar**
- Initial Folder Structure using Atomic Design
- Sign up / login forms

image

### CODE REVIEW

File: src/components/organisms/loginForm/LoginForm.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import Form from '../../atoms/form'
+import LabelInput from '../../molecules/labelInput';
+import {MailOutlined, LockOutlined} from '../../atoms/icon'
+import Button, { BUTTON_SIZES } from '../../atoms/button'
+import style from './loginForm.module.css';
+
+const LoginForm = (props) => {
+    const formLayout = {
+        labelCol:{
```
Comment (sj210) [inline src/components/organisms/loginForm/LoginForm.js:9]: What is this for ?

File: src/components/organisms/signupForm/SignupForm.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import Form from '../../atoms/form'
+import LabelInput from '../../molecules/labelInput';
+import {MailOutlined, LockOutlined} from '../../atoms/icon'
+import Button, { BUTTON_SIZES } from '../../atoms/button'
+import style from './signupForm.module.css';
+import { Input } from 'antd';
+
+const SignupForm = (props) => {
+    const formLayout = {
+        labelCol:{
```
Comment (sj210) [inline src/components/organisms/signupForm/SignupForm.js:10]: what is this for?

**Review by sj210 — APPROVED** [review]

File: src/components/organisms/loginForm/LoginForm.js (hunk 1 above)
Comment (Vageshwar) [reply src/components/organisms/loginForm/LoginForm.js:9]: The main idea was to add a feature for responsive forms : the selected piece of code changes the layout of the form from horizontal to vertical
It isn't used though

File: src/components/organisms/signupForm/SignupForm.js (hunk 2 above)
Comment (Vageshwar) [reply src/components/organisms/signupForm/SignupForm.js:10]: The main idea was to add a feature for responsive forms : the selected piece of code changes the layout of the form from horizontal to vertical
It isn't used though

**Pull request merged**